sim/src/main/scala/midasexamples/Config.scala

```diff
@@ -19,7 +19,11 @@ class DefaultF1Config extends Config(new Config((site, here, up) => {
   case EnableModelMultiThreading => true
   case SynthPrints => true
   case EnableAutoCounter => true
-}) ++ new Config(new firesim.configs.WithEC2F1Artefacts ++ new WithDefaultMemModel ++ new midas.F1Config))
+}) ++ new Config(
+  new firesim.configs.WithEC2F1Artefacts ++
+  new WithDefaultMemModel ++
+  new firesim.configs.WithILATopWiringTransform ++
```

Comment:
Are you intentionally sneaking `WithILATopWiringTransform` back in here?  Should it really be a part of this PR?